test/backend/entity_backend_test.gd

```diff
@@ -681,6 +681,78 @@ func test_check_properties_will_change_after_move() -> void:
 	assert_that(will_change3).is_false()
 	assert_that(will_change4).is_true()
 
+func test_import_duplicates() -> void:
```

Comment:
Just a thought: we should probably move these tests into api_test, as there the testdata stuff is already set up. The backend tests here are supposed to test backend only, not the Pandora singleton.